tests/terraform/graph/resources/arrays/main.tf

```diff
@@ -0,0 +1,46 @@
+resource "x" "pass1" {
+  arr = [
+    "allowed1"
+  ]
+}
+
+resource "x" "pass2" {
+  arr = [
+  ]
+}
```

Comment:
not sure about this one, is no value a subset of a list of values? mathematically it is correct, but does it make sense for the user? @JamesWoolfenden any thoughts?

Reply:
Well, it's either this or return a mathematically incorrect value